crates/phala-types/src/lib.rs

```diff
@@ -328,8 +328,22 @@ pub mod messaging {
         },
     }
 
+    #[cfg(feature = "std")]
+    impl MessageHashing for MiningReportEvent {
```

Comment:
Let add some comment like: We have skipped the iteration here because it's not stable when replaying.